Corona-Warn-App/src/main/res/values-ar/strings.xml

```diff
@@ -0,0 +1,1024 @@
+    <!-- XBUT: risk card - activate tracing -->
+    <string name="risk_card_button_enable_tracing">"تنشيط تسجيل المخالطة"</string>
+    <!-- XTXT: risk card - tracing is off, user should activate to get an updated risk level -->
+    <string name="risk_card_body_tracing_off">"قم بتنشيط تسجيل المخالطة لتقييم حالة الخطر لديك اليوم."</string>
+    <!-- XHED: risk card - low risk headline -->
+    <string name="risk_card_low_risk_headline">"خطر منخفض"</string>
+    <!-- XHED: risk card - increased risk headline -->
+    <string name="risk_card_increased_risk_headline">"خطر أعلى"</string>
+    <!-- XTXT: risk card - increased risk days since last contact -->
+    <plurals name="risk_card_increased_risk_body_contact_last">
+        <item quantity="one">"%1$s يوم منذ آخر تجمُّع"</item>
+        <item quantity="other">"%1$s من الأيام منذ آخر تجمُّع"</item>
+    </plurals>
+    <!-- XHED: risk card - unknown risk headline -->
+    <string name="risk_card_unknown_risk_headline">"خطر غير معروف"</string>
+    <!-- XTXT: risk card - tracing isn't active long enough, so a new risk level can't be calculated -->
+    <string name="risk_card_unknown_risk_body">"تعذر علينا تقدير خطر العدوى نظرًا لعدم قيامك بتنشيط تسجيل المخالطة لفترة طويلة بما فيه الكفاية."</string>
+    <!-- XHED: risk card - tracing stopped headline, due to no possible calculation -->
+    <string name="risk_card_no_calculation_possible_headline">"تسجيل المخالطة توقف"</string>
+    <!-- XTXT: risk card - last successfully calculated risk level -->
+    <string name="risk_card_no_calculation_possible_body_saved_risk">"آخر تسجيل مخالطة:"<xliff:g id="line_break">"\n"</xliff:g>"%1$s"</string>
+    <!-- XHED: risk card -  outdated risk headline, calculation isn't possible -->
+    <string name="risk_card_outdated_risk_headline">"تسجيل المخالطة غير ممكن"</string>
+    <!-- XTXT: risk card - outdated risk, calculation couldn't be updated in the last 24 hours -->
+    <string name="risk_card_outdated_risk_body">"تعذر تحديث تسجيل المخالطة لأكثر من 24 ساعة."</string>
+    <!-- XHED: risk card - loading headline, update is ongoing -->
+    <string name="risk_card_loading_headline">"الفحص قيد التشغيل..."</string>
+    <!-- XTXT: risk card - loading, data is downloaded and will be checked, this can take several minutes -->
+    <string name="risk_card_loading_body">"يتم تنزيل البيانات الحالية وفحصها.\nقد يستغرق هذا عدة دقائق."</string>
+
+    <!-- ####################################
+                    Main
+    ###################################### -->
+
+    <!-- XHED: main, FAQ -->
+    <string name="main_about_headline">"الأسئلة المتكررة"</string>
+    <!-- XTXT: main, explains faq on card -->
+    <string name="main_about_body">"يمكنك أن تجد هنا إجابات الأسئلة المتكررة حول تطبيق Corona-Warn-App. سيتم توجيهك إلى موقع إلكتروني خارجي."</string>
+    <!-- XTXT: FAQ link, should be translated  -->
+    <string name="main_about_link">"https://www.bundesregierung.de/corona-warn-app-faq"</string>
```

Comment:
there is an Arabic FAQ: https://www.bundesregierung.de/breg-de/themen/corona-warn-app/corona-warn-app-arabisch

so maybe
```suggestion
    <string name="main_about_link">"https://www.bundesregierung.de/breg-de/themen/corona-warn-app/corona-warn-app-arabisch"</string>
```